eox_tagging/test/test_viewset.py

```diff
@@ -97,6 +146,36 @@ def test_filter_by_username(self):
         response = self.client.get(URL)
         self.assertEqual(response.status_code, 200)
 
+    def test_filter_by_owner_user(self):
+        """Used to test getting a tag given its owner of type user."""
+        URL = "/api/v1/tags/?owner_type=user"
```

Comment:
reverse and you don't need to create the url with the query params you can do the following https://docs.djangoproject.com/en/3.0/topics/testing/tools/#django.test.Client.get

Reply:
fixed

Reply:
what did you fix ?

Reply:
I added the reverse. Sorry I didn't specify